### PR TITLE
Front page improvements

### PR DESCRIPTION
**Pull request by lbarcziova**
The screenshots inside sections of the image describing the workflow are clickable (screenshot in normal size is opened).

Followups:
- examples, something like [this](https://lark-parser.readthedocs.io/en/latest/examples/index.html)
- real-time statistics (e.g.number of builds this week/today)

Preview (you can also run `hugo server` in this branch to try it out):
Snímka obrazovky z 2022-03-18 08-00-15

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -4,27 +4,65 @@ date: 2019-06-28
...
-1. [packit]({{< ref "cli" >}}), a CLI tool, which you can
+1. a [CLI tool]({{< ref "cli" >}}), which you can
    [install locally](/docs/guide/#have-packit-tooling-installed-locally)
-   and start using right away.
+   and start using right away,
 
-2. [Packit Service]({{< ref "packit-service" >}}), can provide you feedback
-   on how is your project integrated with Fedora Linux or CentOS Stream.
+2. a [service]({{< ref "packit-service" >}}), which provides you feedback
+   on how is your project integrated with [Fedora Linux](https://getfedora.org) or [CentOS Stream](https://www.centos.org/centos-stream/).
```
Comment (TomasTomecek) [inline content/_index.md:16]:
```suggestion
   on how is your project integrated with [Fedora Linux](https://getfedora.org), [CentOS Stream](https://www.centos.org/centos-stream/) and other distributions.
```

Copr supports much more :)

File: content/_index.md [hunk 2]
```diff
@@ -4,27 +4,65 @@ date: 2019-06-28
...
+   <--->
 
-Feel free to jump into [the guide]({{< ref "guide" >}}) for using packit.
+### Who should use Packit?
 
-### Contact
+Any upstream maintainer who would like to automate validation, testing or releasing of code changes
+in his project(s).
```
Comment (TomasTomecek) [inline content/_index.md:22]:
```suggestion
Upstream maintainers who would like to automate validation, testing and releasing of code changes
in downstream distributions.
```
I mainly wanted to make this gender-neutral.

File: content/_index.md [hunk 3]
```diff
@@ -4,27 +4,65 @@ date: 2019-06-28
...
+#### Testing
+
+Packit runs your tests in [Testing Farm](https://docs.testing-farm.io/general/0.1/index.html) and can test either the RPM package built in Copr or
+trigger tests directly on different platforms.
+<--->
+
+#### Automating Fedora releases
+
+With Packit, you can easily
+get the new upstream releases to the [Fedora dist-git](https://src.fedoraproject.org), automatically submit builds in [Koji](https://koji.fedoraproject.org/koji/), and we are currently
```
Comment (TomasTomecek) [inline content/_index.md:45]:
```suggestion
get your upstream releases to the [Fedora dist-git](https://src.fedoraproject.org), automatically submit builds in [Koji](https://koji.fedoraproject.org/koji/), and we are currently
```

File: content/_index.md [hunk 4]
```diff
@@ -4,27 +4,65 @@ date: 2019-06-28
...
+
+### How does Packit look like in action?
+
+![Packit workflow](/images/build_test_workflow.jpg)
+
+### Why start using Packit?
+
+1. _Easy to configure_ - Feel free to jump into [our guide]({{< ref "guide" >}}) or check directly the [configuration documentation](/docs/configuration).
+
+2. _Stable users_ - Systemd, cockpit, avocado, ansible-lint and many other projects. If you are interested,
```
Comment (TomasTomecek) [inline content/_index.md:57]: could you please make these links to the github projects?

**Review by TomasTomecek — COMMENTED** [review]
It looks great!!!

**Comment by xsuchy** [comment]
What is packit? - is service **and** cli. So - everything. The main purpose is to be a placeholder with link to pages with more documentation about cli/service. Users are interesting in this part only after they know what packit is and why should they use it. I suggest renaming this section to "Parts of Packit" and moving it down below the "Why start using Packit".

**Comment by nikromen** [comment]
> What is packit? - is service and cli. So - everything.

I think that we discussed this on arch that we would like to see us not speaking of packit-service but call it only packit (as a whole) since packit-service is just our implementation detail

File: content/_index.md [hunk 5]
```diff
@@ -4,27 +4,66 @@ date: 2019-06-28
...
+#### Testing
+
+Packit runs your tests in [Testing Farm](https://docs.testing-farm.io/general/0.1/index.html) and can test either the RPM package built in Copr or
+trigger tests directly on different platforms.
+<--->
+
+#### Automating Fedora releases
+
+With Packit, you can easily
+get your upstream releases to the [Fedora dist-git](https://src.fedoraproject.org), automatically submit builds in [Koji](https://koji.fedoraproject.org/koji/), and we are currently
```
Comment (xsuchy) [inline content/_index.md:46]: Do we have this actually somewhere documented (from top view)? I am thinking of link "learn more".

File: content/_index.md [hunk 6]
```diff
@@ -4,27 +4,66 @@ date: 2019-06-28
...
+
+### How does Packit look like in action?
+
+![Packit workflow](/images/build_test_workflow.jpg)
+
+### Why start using Packit?
+
+1. _Easy to configure_ - Feel free to jump into [our guide]({{< ref "guide" >}}) or check directly the [configuration documentation](/docs/configuration).
+
+2. _Stable users_ - [Systemd](https://github.com/systemd/systemd/), [cockpit](https://github.com/cockpit-project/cockpit), [avocado](https://github.com/avocado-framework/avocado), [ansible-lint](https://github.com/ansible-community/ansible-lint) and many other projects. If you are interested,
```
Comment (xsuchy) [inline content/_index.md:58]: Hmm, should we link the main page of these projects - or should we link directly to packit.yaml in their git repo?

File: content/_index.md (hunk 5 above)
Comment (lachmanfrantisek) [reply content/_index.md:46]: The "get your upstream releases" is documented as `propose-downstream` job and `Koji` builds will be available once we merge #403. (We can add the link later on.)

And the whole overview can be seen here #404. (We are discussing [here](https://github.com/packit/packit.dev/pull/404#discussion_r826630151) about reusing/updating this to be the overall guide or creating a new, similar, text that will cover this.

File: content/_index.md [hunk 7]
```diff
@@ -4,27 +4,66 @@ date: 2019-06-28
...
+   on how is your project integrated with [Fedora Linux](https://getfedora.org), [CentOS Stream](https://www.centos.org/centos-stream/)
+   and other distributions.
+   <--->
 
-Feel free to jump into [the guide]({{< ref "guide" >}}) for using packit.
+### Who should use Packit?
 
-### Contact
+Upstream maintainers who would like to automate validation, testing and releasing of code changes
+in downstream distributions.
```
[the remark this answers is not in the document]
Comment (TomasTomecek) [reply content/_index.md:23]: sounds good, I would reword it to:

"Upstream projects that want to build RPM packages out of pull requests, commits or releases and bring those releases to Fedora Linux."

File: content/_index.md (hunk 6 above)
Comment (TomasTomecek) [reply content/_index.md:58]: I was also thinking about linking to packit.yaml but... we'd need to make sure they don't rename it or move around (or even drop 😩), so the conclusion in my head was to link to the github project and let people "discover" how they use packit (open a PR, click on checks, look at packit.yaml)

Comment (lbarcziova) [reply content/_index.md:58]: We could still use permalinks to the current versions of their packit configs, but that could get outdated as well, so I am also not sure :confused:

File: content/_index.md (hunk 5 above)
Comment (lbarcziova) [reply content/_index.md:46]: I would probably add the link to the whole overview when it will be merged (not sure about linking the specific jobs, since they should be able to reach from the overview and the previous `features` also do not directly link the jobs.

File: content/_index.md [hunk 8]
```diff
@@ -4,27 +4,66 @@ date: 2019-06-28
...
 
-- **#packit** on [Libera.Chat](https://libera.chat/)
-- **hello@packit.dev**
+{{< columns >}}
+
+#### Building RPMs
+
+Packit uses [Copr build system](https://copr.fedorainfracloud.org/) to build RPMs from your code changes,
+so you can validate your package will be correctly built from each pull request or have automation around your Copr
+repository containing builds from your upstream releases/commits.
```
[the remark this answers is not in the document]
Comment (lbarcziova) [reply content/_index.md:34]: Yeah, I struggled with this sentence :D I will try to reword it considering your suggestion.

File: content/_index.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (lbarcziova) [reply content/_index.md:23]: Thanks for the suggestions! I agree with mentioning building RPM packages and changing upstream maintainers to upstream projects.
> "Upstream projects that want to build RPM packages out of pull requests, commits or releases and bring those releases to Fedora Linux."

This is nice and clear, but I personally am missing the testing part, so I would adjust it a little bit, WDYT?

File: content/_index.md [hunk 9]
```diff
@@ -4,27 +4,65 @@ date: 2019-06-28
...
+
+Packit runs your tests in [Testing Farm](https://docs.testing-farm.io/general/0.1/index.html) and can test either the RPM package built in Copr or
+trigger tests directly on different platforms.
+<--->
+
+#### Automating Fedora releases
+
+With Packit, you can easily
+get the new upstream releases to the [Fedora dist-git](https://src.fedoraproject.org), automatically submit builds in [Koji](https://koji.fedoraproject.org/koji/), and we are currently
+finalizing implementation for automatic creation of [Bodhi](https://bodhi.fedoraproject.org) updates!
```
[the remark this answers is not in the document]
Comment (lbarcziova) [reply content/_index.md:47]: :+1: will reword

File: content/_index.md (hunk 6 above)
Comment (TomasTomecek) [reply content/_index.md:58]: maybe it's finally time to start checking URLs in our docs if they are up to date :D

anyway, I'm +1 to link directly the configs (since that may be the first thing people look for)

Comment (xsuchy) [reply content/_index.md:58]: > We could still use permalinks to the current versions of their packit configs, but that could get outdated as well, so I am also not sure confused

We can link to specific treeish. E.g.,  https://github.com/systemd/systemd/blob/d15e1a29e3aab04ee79d5e3ec8e1e65fca78e165/.packit.yml But this, on the other hand, does not get updates when we update some of our API (format of packit.yaml), which will be incompatible with the current format. Hmm, but that may or may not happen. Definitely in the distant future. :)

Comment (lbarcziova) [reply content/_index.md]: > We can link to specific treeish. E.g., https://github.com/systemd/systemd/blob/d15e1a29e3aab04ee79d5e3ec8e1e65fca78e165/.packit.yml 

Yes, that's what I meant by permalinks.

Agreed, changed it to the links to configs.

**Comment by lbarcziova** [comment]
I think this can be the final version of this PR (and I will open issues with the followups in the description) - the screenshots inside the image describing workflow are now clickable, What is Packit section reorganized and Get started button added. PTAL

**Review by xsuchy — APPROVED** [review]
+1 Thank you.

**Review by lachmanfrantisek — APPROVED** [review]
Thanks a lot for working on this.

We can easily tweak that later but this is definitely a huge improvement from what we had before!

**Review by jpopelka — APPROVED** [review]
:rocket:
